src/analysis/utils/runtime-utils.ts

```diff
@@ -74,15 +75,15 @@ export type NormalizedComputeRegion = NominalType<string, 'ComputeRegion'>;
 // TODO: test when zone and region have types
 export const getNormalizedComputeRegion = (config: GoogleRuntimeConfig | AzureConfig): NormalizedComputeRegion => {
   if (isGceConfig(config) || isGceWithPdConfig(config)) {
-    return getRegionFromZone(config.zone).toUpperCase() as NormalizedComputeRegion;
+    return (getRegionFromZone(config.zone) || defaultComputeRegion).toUpperCase() as NormalizedComputeRegion;
```

Comment:
`GceConfig` and `GceWithPdConfig` both have `zone` as nonoptional. Are you concerned about a case where it's coming in as an empty string?

Reply:
resolved in person, addressed the model discrepancies and put defaults where necessary